server/room-battle.js

```diff
@@ -997,6 +997,13 @@ class RoomBattle extends RoomGames.RoomGame {
 		return player;
 	}
 
+	forcedPublic() {
+		for (const userid in this.playerTable) {
+			const user = Users(userid);
+			if (user && user.forcedPublic) return user.forcedPublic;
```

Comment:
This is potentially all overkill - we might never have/need more than one 'forced public' prefix at the same time, and maybe we don't even care about reporting what the prefix is to users - I just figured the extra explanation would be helpful given we're kind of springing this privacy regression on them,

Reply:
I think it's fine. I don't think it's 100% needed but it doesn't do harm, and is a nice addition.